jsons/Policies.json

```diff
@@ -501,7 +501,7 @@
 			{
 				"name": "Skyscrapers",
 				"uniqueText": "Cost of purchasing buildings reduced by [50]%",
-				"uniques": ["Cost of purchasing [Science] buildings reduced by [50]%","Cost of purchasing [Production] buildings reduced by [50]%","Cost of purchasing [Food] buildings reduced by [50]%","Cost of purchasing [Gold] buildings reduced by [50]%","Cost of purchasing [Happiness] buildings reduced by [50]%","Cost of purchasing [Culture] buildings reduced by [50]%"],
+				"uniques": ["[-50]% Gold Cost of purchasing [Non-Military] Buildings"]
```

Comment:
I dont think "Non-Military" is a valid buildingfilter

Reply:
List of valid ones: https://github.com/yairm210/Unciv/blob/master/docs/unique%20parameters.md#buildingfilter

Reply:
Dunno how to fix it. Can you suggest?

Reply:
Depends on what you want this unique to do. If you want it to reduce the cost of purchasing all buildings, you'd need `[Gold] cost of purchasing [All] buildings [-50]%`, while if you only want to decrease the cost of buildings giving stats, you'd need the old uniques.